src/OVAL/public/oval_definitions.h

```diff
@@ -1828,15 +1901,15 @@ struct oval_component *oval_variable_get_component(struct oval_variable *);	//ty
  * It should be freed after use by the calling application.
  * @memberof oval_variable
  */
-struct oval_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
+struct oval_variable_possible_value_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
```

Comment:
I guess if we wanted to maintain API compatibility AND fix python we'd have to expose the plain iterator functions.